integration/poller/poller_test.go

```diff
@@ -4,8 +4,10 @@ import (
 	"bytes"
 	"context"
 	"crypto/rand"
+	mathrand "math/rand"
```

Comment:
Probably not very relevant at the moment but I think we can already use [v2](https://pkg.go.dev/math/rand/v2)

Reply:
Looks like it supports the use case, so I've pushed a commit for this.